presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -1054,7 +1055,12 @@ public SystemSessionProperties(
                         VERBOSE_EXCEEDED_MEMORY_LIMIT_ERRORS_ENABLED,
                         "When enabled the error message for exceeded memory limit errors will contain additional operator memory allocation details",
                         nodeMemoryConfig.isVerboseExceededMemoryLimitErrorsEnabled(),
-                        false));
+                        false),
+                booleanProperty(
+                        QUERY_OPTIMIZATION_WITH_MATERIALIZED_VIEW_ENABLED,
+                        "Enable query optimization with materialized view",
+                        featuresConfig.isQueryOptimizationWithMaterializedViewEnabled(),
+                        true));
```

Comment:
Does this enable the feature for all queries by default? 
I assume enabling it will introduce some overhead for queries which cannot use materialize views as well, e.g. additional requests to metastore to fetch MV info after https://github.com/prestodb/presto/issues/16430 is addressed?
Should we disable it by default and figure out how to rollout the change safely?

Reply:
> Does this enable the feature for all queries by default?
> I assume enabling it will introduce some overhead for queries which cannot use materialize views as well, e.g. additional requests to metastore to fetch MV info after #16430 is addressed?
> Should we disable it by default and figure out how to rollout the change safely?

It is disabled by default.